airflow/providers/databricks/hooks/databricks.py

```diff
@@ -519,13 +519,27 @@ def delete_run(self, run_id: int) -> None:
         json = {"run_id": run_id}
         self._do_api_call(DELETE_RUN_ENDPOINT, json)
 
-    def repair_run(self, json: dict) -> None:
+    def repair_run(self, json: dict) -> int:
         """
         Re-run one or more tasks.
 
         :param json: repair a job run.
         """
-        self._do_api_call(REPAIR_RUN_ENDPOINT, json)
+        response = self._do_api_call(REPAIR_RUN_ENDPOINT, json)
+        return response["repair_id"]
+
+    def get_latest_repair_id(self, run_id: int) -> Optional[int]:
```

Comment:
```suggestion
    def get_latest_repair_id(self, run_id: int) -> int | None:
```

Reply:
should I change this @pankajastro  ?

Reply:
Yep, this is a part of fixing of [Static Code Checks](https://github.com/apache/airflow/blob/main/STATIC_CODE_CHECKS.rst)

```console
airflow/providers/databricks/hooks/databricks.py:531:52: UP007 Use `X | Y` for type annotations
```